src/cookies.test.ts

```diff
@@ -95,6 +105,33 @@ describe('Cookies', () => {
                 cookieOptions,
             );
         });
+
+        // TODO: update test when PECR purposes introduced
+        it('is in preview', () => {
+            Object.defineProperty(document, 'domain', {
+                value: 'viewer.gutools.co.uk',
+                configurable: true,
+            });
+            global.guardian = {
```

Comment:
Do you need this `global.guardian` anymore?

Reply:
I do not. Cleaned.